flink/README.md

```diff
@@ -0,0 +1,83 @@
+# Agent Check: flink
+
+## Overview
+
+This check monitors [Flink][1]. Datadog collects Flink metrics through Flink's
+[Datadog HTTP Reporter][4], which uses [Datadog's HTTP API][2].
+
+## Setup
+
+### Installation
+
+The Flink check is included in the [Datadog Agent][3] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Configure the [Datadog HTTP Reporter][4] in Flink.
+
+Copy `<FLINK_HOME>/opt/flink-metrics-datadog-1.9.2.jar` into your `<FLINK_HOME>/lib` folder.
+In your `<FLINK_HOME>/conf/flink-conf.yaml`, add these lines, replacing `xxx` with your Datadog API key:
+
+```yaml
+metrics.reporter.dghttp.class: org.apache.flink.metrics.datadog.DatadogHttpReporter
+metrics.reporter.dghttp.apikey: xxx
+```
+
+2. Re-map system scopes in your `<FLINK_HOME>/conf/flink-conf.yaml`.
+
+```yaml
+metrics.scope.jm: flink.jobmanager
+metrics.scope.jm.job: flink.jobmanager.job
+metrics.scope.tm: flink.taskmanager
+metrics.scope.tm.job: flink.taskmanager.job
+metrics.scope.task: flink.task
+metrics.scope.operator: flink.operator
+```
+
+Note that this step is optional for collecting metrics, but required if you want
+your Flink metrics to be supported and not custom.
+
+3. Add any [tags][4] you may want in your `<FLINK_HOME>/conf/flink-conf.yaml`.
+Here is an example of custom tags:
+
+```yaml
+metrics.reporter.dghttp.tags: host:tag1,tag2
```

Comment:
does this mean a tag could look like `host:tag1` and `tag2` or `host:tag1` and `host:tag2`?

our config tag placeholders look like this https://github.com/DataDog/integrations-core/blob/master/active_directory/datadog_checks/active_directory/data/conf.yaml.example#L55-L57

```suggestion
metrics.reporter.dghttp.tags: <KEY1>:<VALUE1>, <KEY2>:<VALUE2>,
```

Reply:
This means that a tag can look like: `<KEY1>:<VALUE1>` or `<VALUE2>`.

Reply:
nice, let's just keep officially documented type `<KEY1>:<VALUE1>` since our tags also support both of the above. See https://docs.datadoghq.com/tagging/#defining-tags (#4 specifically)

Reply:
Sounds good, updated!